roles/kube_proxy/files/kube-proxy.yaml

```diff
@@ -0,0 +1,150 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: proxy
+  namespace: kube-proxy
+  annotations:
+    kubernetes.io/description: |
+      This daemonset launches kube-proxy.
+    image.openshift.io/triggers: |
+      [
+        {"from":{"kind":"ImageStreamTag","name":"node:v3.10"},"fieldPath":"spec.template.spec.containers[?(@.name==\"proxy\")].image"}
```

Comment:
Its best not to hardcode version instead something like this should do the trick s/v3.10/${version}/